docs/en/operations/system-tables/schema_inference_cache.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: /en/operations/system-tables/schema_inference_cache
+---
+# Schema inference cache
+
+Contains information about all cached file schemas.
+
+Columns:
+- `storage` ([String](../../sql-reference/data-types/string.md)) — Storage name: File, URL, S3 or URL.
+- `source` ([String](../../sql-reference/data-types/string.md)) — File source.
+- `format` ([String](../../sql-reference/data-types/string.md)) — Format name.
+- `additional_format_info` ([String](../../sql-reference/data-types/string.md)) - Additional information required to identify the schema. For example, format specific settings.
+- `registration_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — Timestamp when schema was added in cache.
+- `schema` ([String](../../sql-reference/data-types/string.md)) - Cached schema.
+
+**Example**
+
```

Comment:
I will add a line around here to have the reader pop the file into their user_data dir and then open clickhouse-client to run the commands.